multi-cluster-services/multi-cluster-services-basic/README.md

```diff
@@ -0,0 +1,195 @@
+# Multi-cluster Services
+
+[Multi-cluster Services](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services) for GKE enables existing Services to be discoverable and accessible across clusters with a virtual IP, matching the behavior of a [ClusterIP Service](https://cloud.google.com/kubernetes-engine/docs/concepts/service#services_of_type_clusterip)
+accessible in a cluster.
+
+### Use-cases
+
+- High availability by running the same service across clusters
+- Flexible migration between clusters
+
+### Relevant documentation
+
+- [Multi-cluster Services Concepts](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services)
+- [Setting Up Multi-cluster Services](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services)
+- [OSS Multi-cluster Services API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
+
+#### Versions
+
+- GKE clusters on GCP
+- 1.17 and later versions of GKE supported
+- Tested and validated with 1.18.10-gke.1500 on Feb 15th 2021
+
+### Networking Manifests
+
+This recipe demonstrates deploying Multi-cluster Services in a cluster (`gke-1`) and make it accessible to other cluster (`gke-2`). In later steps, it also demonstrates app migration to a new cluster (`gke-2`) by exporting the same Service from `gke-2`. The exported Services can be accessed via DNS name `$svc.$ns.svc.clusterset.local` or a VIP called `ClusterSetIP`.
+
+![basic multi-cluster services](../../images/multi-cluster-services.png)
```

Comment:
This image is a little confusing because it doesn't really match the recipe. It seems to suggest that there are three clusters involved here. 

I put together a diagram here. Feel free to edit it and use it: https://docs.google.com/presentation/d/1Wngda7LN4GcMpASvdnG-laLUDOt3hzmPeUuVvMdSXA0/edit#slide=id.gd0d19b874a_0_0

Reply:
Thanks for the diagram. I updated the image based on your diagram.